modules/odc-uninstalling-web-terminal.adoc

```diff
@@ -17,7 +17,12 @@ Uninstalling the Web Terminal Operator does not remove any of its custom resourc
 
 == Deleting the web terminal components and custom resources
 
-Use the CLI to delete the CRs that are created during installation of the  Web Terminal Operator.
+Use the CLI to delete the CRs that are created during installation of the Web Terminal Operator.
```

Comment:
https://deploy-preview-33764--osdocs.netlify.app/openshift-enterprise/latest/web_console/odc-about-web-terminal.html
we need to update the suggested channel here
![Screenshot_20210630_181204](https://user-images.githubusercontent.com/5887312/123986119-b4846d00-d9ce-11eb-9987-df48bba97f76.png)

Reply:
Done (squashed into first commit in this PR)